packages/core/src/fields/types/multiselect/index.ts

```diff
@@ -0,0 +1,210 @@
+import inflection from 'inflection';
+import { humanize } from '../../../lib/utils';
+import {
+  BaseListTypeInfo,
+  FieldTypeFunc,
+  CommonFieldConfig,
+  FieldData,
+  jsonFieldTypePolyfilledForSQLite,
+} from '../../../types';
+import { graphql } from '../../..';
+import { assertCreateIsNonNullAllowed, assertReadIsNonNullAllowed } from '../../non-null-graphql';
+import { resolveView } from '../../resolve-view';
+import { userInputError } from '../../../lib/core/graphql-errors';
+
+export type MultiselectFieldConfig<ListTypeInfo extends BaseListTypeInfo> =
+  CommonFieldConfig<ListTypeInfo> &
+    (
+      | {
+          /**
+           * When a value is provided as just a string, it will be formatted in the same way
+           * as field labels are to create the label.
+           */
+          options: readonly ({ label: string; value: string } | string)[];
+          /**
+           * If `enum` is provided on SQLite, it will use an enum in GraphQL but a string in the database.
+           */
+          type?: 'string' | 'enum';
+          defaultValue?: readonly string[];
+        }
+      | {
+          options: readonly { label: string; value: number }[];
+          type: 'integer';
+          defaultValue?: readonly number[];
+        }
+    ) & {
+      graphql?: {
+        create?: {
+          isNonNull?: boolean;
+        };
+        read?: {
+          isNonNull?: boolean;
+        };
+      };
+      db?: {
+        map?: string;
+      };
+    };
+
+// These are the max and min values available to a 32 bit signed integer
+const MAX_INT = 2147483647;
+const MIN_INT = -2147483648;
+
+export const multiselect =
+  <ListTypeInfo extends BaseListTypeInfo>({
+    ui,
+    defaultValue = [],
+    ...config
+  }: MultiselectFieldConfig<ListTypeInfo>): FieldTypeFunc<ListTypeInfo> =>
+  meta => {
+    if ((config as any).isIndexed === 'unique') {
+      throw Error("isIndexed: 'unique' is not a supported option for field type password");
```

Comment:
Should this be 'field type multiselect'?

Reply:
```suggestion
      throw Error("isIndexed: 'unique' is not a supported option for field type multiselect");
```